go/vt/wrangler/traffic_switcher.go

```diff
@@ -1176,6 +1176,13 @@ func (ts *trafficSwitcher) changeTableSourceWrites(ctx context.Context, access a
 		}
 		return err
 	})
+	if err != nil {
+		log.Warningf("Error in changeTableSourceWrites: %s", err)
+		return err
+	}
+	// Note that the denied tables, which are being updated in this method, are not part of the SrvKeyspace. However we are using the notification of
+	// a SrvKeyspace change in VTGate to recompute the state of a MoveTables workflow. So we need to trigger a SrvKeyspace change here.
+	return ts.TopoServer().RebuildSrvVSchema(ctx, nil)
```

Comment:
the comment says about trigger `SrvKeyspace` and the call is to `SrvVSchema`. Also, will the call create an updated struct with denied tables?

Reply:
Fixed naming.  In the relevant vtgate code we also get denied tables (which are in the shard record) when we get notified of a changed SrvVSchema.